build.sbt

```diff
@@ -130,30 +130,32 @@ val benchmarks = project
     skip in publish := true,
   )
 
+val integrations = file("integrations")
```

Comment:
@jvican do you want me to move this to BuildPlugin as well?

Reply:
No, this is okay here